requirements.txt

```diff
@@ -1,13 +1,13 @@
 yq
 pyyaml
 fire
-ansible==2.9.*
-jinja2==3.0.*
+ansible
+jinja2
 flake8
 pylint
 awscli
 numpy
 joblib
 
 jsonpath_ng
-state-signals==0.5.2
```

Comment:
@kpouget quick question, is there a need for the pins on those packages versions? ansible, jinja2, and state-signals?

Reply:
as long as nothing breaks, not really